src/components/BottomBar/CurrentTask.js

```diff
@@ -1,33 +1,32 @@
 import { observer } from 'mobx-react';
-import { useMemo } from 'react';
 import { Button } from '../../common/Button/Button';
 import { Block, Elem } from '../../utils/bem';
 import { guidGenerator } from '../../utils/unique';
 import { isDefined } from '../../utils/utilities';
+import { Tooltip } from '../../common/Tooltip/Tooltip';
 import './CurrentTask.styl';
 
 
 export const CurrentTask = observer(({ store }) => {
-  const currentIndex = useMemo(() => {
-    return store.taskHistory.findIndex((x) => x.taskId === store.task.id) + 1;
-  }, [store.taskHistory]);
 
   const historyEnabled = store.hasInterface('topbar:prevnext');
   // @todo some interface?
   const canPostpone = !isDefined(store.annotationStore.selected.pk)
     && !store.canGoNextTask
     && !store.hasInterface('review')
     && store.hasInterface('postpone');
-
+  
   return (
     <Elem name="section">
       <Block name="current-task" mod={{ 'with-history': historyEnabled }}>
         <Elem name="task-id">
           {store.task.id ?? guidGenerator()}
           {historyEnabled && (
-            <Elem name="task-count">
-              {currentIndex} of {store.taskHistory.length}
-            </Elem>
+            <Tooltip name={'task-count'} title={`${store.queuePosition} tasks complete of ${store.queueTotal} total tasks`}>
```

Comment:
The queuePosition isn't the number of tasks complete, its the position within the task queue according to the number of tasks available to the annotator, assigned + could be assigned (if in auto mode).